tui/src/driver.c

```diff
@@ -0,0 +1,95 @@
+#include "driver.h"
+
+#include <stdbool.h>
+#include <stdio.h>
+
+bool init_driver(driver_t *driver, const char *dev_path) {
```

Comment:
At some point I may look into figuring out how to set serial port settings from C. Firmata expects baud rate of 57600, 8 data bits, 1 stop bit, no parity